pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -6645,6 +6952,21 @@ def process_data(self):
             if len(self.leakage_qbnames) == 0:
                 self.leakage_qbnames = None
 
+        # prepare list of qubits on which must be considered simultaneously
+        # for preselection. Default: preselect on all qubits in the gate = ground
+        default_preselection_qbs = defaultdict(list)
+        for qbn in self.qb_names:
+            for gate_qbs in self.gates_list:
+                if qbn in gate_qbs:
+                    default_preselection_qbs[qbn].extend(gate_qbs)
+        preselection_qbs = self.get_param_value("preselection_qbs",
+                                                default_preselection_qbs)
+        self.options_dict.update({"preselection_qbs": preselection_qbs})
```

Comment:
Why is the above extraction not done in the base MultiCZGate_Calib_Analysis? This looks like it should also be added to CZDynamicPhaseAnalysis.